### PR TITLE
Fix ref anim

### PR DESCRIPTION
**Pull request by mattkrick**
@ackernaut here's the branch with the funny looking stacks. to reproduce: in the group phase, group 3+ cards & have 2+ rows of cards, then drag the top one of the group off. while the top one is being drug, the other 2 cards touch the row below em

### CODE REVIEW

File: src/universal/components/ReflectionGroupHeader.js [hunk 1]
```diff
@@ -18,20 +17,19 @@ const GroupHeader = styled('div')(({isExpanded, phaseType}) => ({
...
   width: '100%'
 }))
 
 const TitleAndCount = styled('div')(({isExpanded}) => ({
   alignItems: 'center',
   display: 'flex',
   flexShrink: 1,
-  justifyContent: !isExpanded && 'center',
-  position: 'relative'
+  justifyContent: 'space-between',
```
Comment (mattkrick) [inline src/universal/components/ReflectionGroupHeader.js]: this breaks the expanded view by putting the card count on the far right, which looks strange during the closing animation.
repro: expand 3 card stack in a 5 column layout

**Comment by mattkrick** [comment]
@ackernaut small bug in the reflection group header, other than that LGTM!

**Comment by ackernaut** [comment]
Patching this up now! Thanks @mattkrick

**Comment by ackernaut** [comment]
One last thing:
- [ ] fix TypeScript warning in `DraggableReflectionCard.tsx:136` and `DraggableReflectionCard.tsx:149`

**Pull request merged**